### ISSUE TITLE
Wireframe: About page content

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#4668]
Gather and finalize content from the about pages based on the [wireframe ](https://mozilla.invisionapp.com/share/ZEWXZW3VXJR#/screens/417708362)and [copy doc](https://docs.google.com/document/d/19Y5hNmEU4wix_gCoSvmhMKUk2F1Dht6Nwf1JQAu0lsg/edit#heading=h.jga10kqap7d8).

**Comment by taisdesouzalessa** [comment]
Here is the feedback we requested from the exec team (placing it here for context):

> Who we are -  what we need from you by June 26 : 
> Copy on how to connect our mission, Internet Health, and our Theory of Change (approx 3 short paragraphs).

**Comment by Pomax** [comment]
@kristinashu it sounds like we can close this one?

**Comment by kristinashu** [comment]
Yes, this is still blocked by a content strategy and I don't see it being unblocked too soon.
